demo/api/package.json

```diff
@@ -18,9 +19,9 @@
         "mikro-orm": "mikro-orm",
         "mikro-orm:drop": "mikro-orm schema:drop -r",
         "mikro-orm:migration:generate": "mikro-orm migration:create",
-        "start": "rimraf dist && dotenv -c -- nest start",
-        "start:debug": "rimraf dist && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
-        "start:dev": "rimraf dist && dotenv -c -- nest start --watch --preserveWatchOutput",
+        "start": "$npm_execpath prebuild && dotenv -c secrets -- nest start",
+        "start:debug": "$npm_execpath prebuild && dotenv -c secrets -- nest start --debug --watch --preserveWatchOutput",
+        "start:dev": "$npm_execpath prebuild && $npm_execpath db:migrate && $npm_execpath console createBlockIndexViews && dotenv -c secrets -- nest start --watch --preserveWatchOutput",
```

Comment:
why only add migrate and createBlockIndexViews for start:dev and not the other 2 starts?

why do we even have 3?

Reply:
> why do we even have 3?

They are probably here since creating the API with `nest new`. We may remove them I suppose.